nncf/openvino/quantization/quantize_model.py

```diff
@@ -366,6 +369,18 @@ def quantize_impl(
     )
 
 
+@tracked_function(
+    NNCF_OV_CATEGORY,
+    [
+        CompressionStartedWithCompressWeightsApi(),
```

Comment:
I would extend it by 'all_layers'. cc @AlexKoff88

Reply:
What about `ratio` and `group_size`?

Reply:
> What about `ratio` and `group_size`?

How do you plan to use that information?

Reply:
It might show how much the ratio and group size have been reduced in order to obtain an accurate model, as an example.

Reply:
Though for gs it's still ok as I think most of users will have 64 or 128

Reply:
Ratio would show how often mixed precision is used, and how often customers choose full int4 model.
Often usage of mixed precision with small ratio and low group size might hint the need for more advanced technics to recover accuracy, as vector quantization, fine-tuning ...

Reply:
Agreed to keep only mode and algo.